test/test_docs.jl

```diff
@@ -1,4 +1,5 @@
 using NBInclude
+import Pkg; Pkg.add("TikzGraphs")
```

Comment:
I think the right way to add packages for tests is through adding `[extras]` and `[targets]` for the test packages similar to what we have [here](https://github.com/JuliaTeX/PGFPlots.jl/blob/master/Project.toml). Travis also seems to be breaking here.